server/example/org.eclipse.glsp.ide.workflow.editor/META-INF/MANIFEST.MF

```diff
@@ -19,12 +20,16 @@ Require-Bundle: org.eclipse.elk.alg.common;bundle-version="0.8.1",
  org.eclipse.elk.graph;bundle-version="0.8.1",
  org.eclipse.lsp4j.websocket;bundle-version="0.11.0",
  org.eclipse.e4.core.contexts;bundle-version="1.8.400",
- org.eclipse.jetty.server;bundle-version="[10.0.0,11.0.0)"
-Bundle-ClassPath: .
-Eclipse-BundleShape: dir
-Bundle-ActivationPolicy: lazy
-Bundle-Activator: org.eclipse.glsp.ide.workflow.editor.Activator
-Bundle-Vendor: Eclipse GLSP
+ org.eclipse.jetty.server;bundle-version="[10.0.0,11.0.0)",
+ org.eclipse.e4.ui.model.workbench;bundle-version="2.2.300.v20221024-2137",
```

Comment:
Do we need to put a specific bundle version here? At least it should be a range imho.

Reply:
Missed this one, that does not really need any version